proguard.pro

```diff
@@ -42,13 +42,14 @@
 -keep class app.lawnchair.LawnchairApp { *; }
 -keep class app.lawnchair.LawnchairLauncher { *; }
 -keep class app.lawnchair.compatlib.** { *; }
+
 -keep class com.google.protobuf.Timestamp { *; }
+
 # TODO: Remove this after the change in https://github.com/ChickenHook/RestrictionBypass/pull/9 has been released.
 -keep class org.chickenhook.restrictionbypass.** { *; }
--keep class androidx.compose.runtime.** { *; }
```

Comment:
Is there any specific reason to keep all classes under this path?

Reply:
Yes, or you can verify building in release?

Reply:
I don't see any crash in Settings screen after removing this line, where you saw?

Reply:
No, I mean you successfully build in release build right? And no crash?. I forgot why I added, but if you don't faced any issue in release build that's great. 

Anyway will check it

Reply:
I checked, nothing crashed, just keep the removing now.